install/cluster-operator/051-Deployment-strimzi-cluster-operator.yaml

```diff
@@ -17,6 +17,10 @@ spec:
         strimzi.io/kind: cluster-operator
     spec:
       serviceAccountName: strimzi-cluster-operator
+      volumes:
```

Comment:
We seem to have this file as 051. And below this a new file as 060 ... should this one be deleted?